testsuite/integration/rts/src/test/config/arq/arquillian.xml

```diff
@@ -23,7 +23,7 @@
 <arquillian xmlns="http://www.jboss.org/arquillian-1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://www.jboss.org/arquillian-1.0 http://jboss.org/schema/arquillian/arquillian-1.0.xsd">
 
-    <defaultProtocol type="jmx-as7" />
+    <defaultProtocol type="Servlet 3.0" />
```

Comment:
please also add comment above this with link to http://discuss.arquillian.org/t/solved-randomly-before-method-runs-twice/193/4 which explains why you are forcing servlet protocol.